services/order.py

```diff
@@ -0,0 +1,34 @@
+from django.db import transaction
+from django.db.models import QuerySet
+
+from db.models import User, Order, Ticket, MovieSession
+
+
+def create_order(
+        tickets: list[dict],
+        username: str,
+        date: str = None
+) -> None:
+
+    with transaction.atomic():
+        order = Order.objects.create(user=User.objects.get(username=username))
+
+        if date:
+            order.created_at = date
+            order.save()
+
+        for ticket in tickets:
+            movie_session = ticket["movie_session"]
+
+            Ticket.objects.create(
+                movie_session=MovieSession.objects.get(id=movie_session),
```

Comment:
bad solution. Read the first item in the checklist:
https://github.com/mate-academy/py-movie-moviesession-cinemahall/blob/master/checklist.md#code-efficiency

Reply:
you shouldn't make one more query to DB. Remember: ForeignKey saves only the primary key to the object, not the full object. By default primary key is an ID. So it's enough to record ID only.